hawc/apps/vocab/api.py

```diff
@@ -12,15 +13,17 @@
 from . import constants, models, serializers
 
 
-class EhvTermViewSet(viewsets.GenericViewSet):
+class VocabTermViewSet(viewsets.GenericViewSet):
```

Comment:
nice job generalizing this. With your initial approach, the data was loaded every time the server restarted, which is all the time in development mode, since it was a class-level property on the class. I changed it to be a callable method, so it's only fetched when needed. I did the same for the view too.